float32[] feature_data
float32[] gesture_data
int32 gesture_height
int32 gesture_width
